projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.ts

```diff
@@ -0,0 +1,366 @@
+                              .pipe(map((spatialReferenceSpecification) => extentToBboxDict(spatialReferenceSpecification.extent))),
+                    of(resultDescriptor.spatialReference),
+                    of(time),
+                ]),
+            ),
+            map(([bbox, crs, time]: [BBoxDict, SrsString, Time]) => {
+                // for sampling, we choose a reasonable resolution
+                const NUM_PIXELS = 1024;
+                const xResolution = (bbox.upperRightCoordinate.x - bbox.lowerLeftCoordinate.x) / NUM_PIXELS;
+                const yResolution = (bbox.upperRightCoordinate.y - bbox.lowerLeftCoordinate.y) / NUM_PIXELS;
+                return {
+                    bbox,
+                    crs,
+                    time: time.toDict(),
+                    spatialResolution: [xResolution, yResolution],
+                };
+            }),
+        );
+
+        return combineLatest([statisticsWorkflow$, queryParams$, this.userService.getSessionOnce()]).pipe(
+            first(),
+            mergeMap(([statisticsWorkflow, queryParams, session]) =>
+                this.backendService.getPlot(statisticsWorkflow, queryParams, session.sessionToken),
+            ),
+            map((plot) => {
+                if (plot.plotType !== 'Statistics') {
+                    throw new Error('Expected `Statistics` plot.');
+                }
+
+                return plot.data as {
+                    [name: string]: {
+                        valueCount: number;
+                        validCount: number;
+                        min: number;
+                        max: number;
+                        mean: number;
+                        stddev: number;
+                    };
+                };
+            }),
+            map((statistics) => {
+                const NUMBER_OF_COLOR_STEPS = 16;
+                const REVERSE_COLORS = false;
+
+                const min = statistics[rasterName].min;
+                const max = statistics[rasterName].max;
+
+                if (min === null || min === undefined || max === null || max === undefined) {
+                    throw new Error('Sample statistics do not have valid min/max values.');
+                }
+
+                const breakpoints = ColorMapSelectorComponent.createLinearBreakpoints(
+                    MPL_COLORMAPS.INFERNO,
+                    NUMBER_OF_COLOR_STEPS,
+                    REVERSE_COLORS,
+                    {
+                        min,
+                        max,
+                    },
+                );
+                const colorizer = new LinearGradient(
+                    breakpoints,
+                    TRANSPARENT,
+                    TRANSPARENT, // TODO: set under/over color to first and last breakpoint when avaialble
+                );
+                return new RasterSymbology(0.8, colorizer);
+            }),
```

Comment:
Maybe you can rather adapt the `geoengine-symbology-creator` component to…

 1. allow disabling the "same as source" option.
 2. Have an option to create a symbology by specifying min and max values (which is currently the fallback here, right?)

Then, you don't need to do the same thing here, but the component gets more versatile.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/546/commits/58814fdd2876a2d61bc13db1f4e7ab5cb23f1d90

Reply:
![image](https://user-images.githubusercontent.com/4947061/208441662-00135923-966e-4bc2-a649-c8b16127c19f.png)

When min=max, it crashes.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/546/commits/7fbdac5051ca3c1be3dea5bc03c9ce8db0bd0ba3

Reply:
![image](https://user-images.githubusercontent.com/4947061/208473976-7b54b5c2-1aee-462b-9db9-99045883e5bb.png)

You can still click create and this causes an error.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/546/commits/54e1d879679d5652e008cad7693110c98cd34556